.deployment.config.json

```diff
@@ -104,7 +112,10 @@
     ]
   },
   "veracode": {
-    "no_scan": true
+    "sandbox_name": "UI Kit",
```

Comment:
Have you tested it/the full loop ?

Sometimes veracode scan can fail for some weird reasons, and it would be extremely annoying to have it block a release.

You could do so by changing the Jenkins job I think, targeting your branch, and to try and release your changes in qa.

Or, have the Veracode scan start in dev.

Reply:
Doc link on how to run it in dev: 

https://coveord.atlassian.net/wiki/spaces/CM/pages/1807876713/System+Certifiers